stat/distuv/distribution_test.go

```diff
@@ -216,6 +216,22 @@ func checkProbQuantContinuous(t *testing.T, i int, xs []float64, c cumulantProbe
 	}
 }
 
+type moder interface {
+	Mode() float64
+}
+
+func checkMode(t *testing.T, i int, xs []float64, m moder, dx float64, tol float64) {
+	rXs := make([]float64, len(xs))
+	for j, x := range xs {
+		rXs[j] = floats.RoundEven(x/dx, 0) * dx
```

Comment:
Please use [`math.RountToEven`](https://golang.org/pkg/math/#RoundToEven). The implementation there is better than ours and ours should be fixed (and even then warned against).

Reply:
Maybe the Gonum one should be removed then? Done.

Reply:
Yeah, maybe. They're different and the philosophy/practicality around them is sort of vexed. The one in math doesn't do precisions other than to integer. There are (entirely valid arguments) that this is because it's not possible to correctly do anything else. But on the other hand people want rounding to arbitrary digits. We have been avoiding looking under that carpet for a while now.

Reply:
Would there be interest in an arbitrary precision module for Gonum?

Reply:
There is math/big for that. There are also decimal packages for arbitrary decimal precision (finance) outside the standard library.